core/vibes/soul/primitives/navigation/index.tsx

```diff
@@ -26,6 +27,7 @@ import { Image } from '~/components/image';
 import { Link } from '~/components/link';
 import { usePathname } from '~/i18n/routing';
 
+import { FormStatus } from '../../form/form-status';
 import { Price } from '../price-label';
 import { ProductCard } from '../product-card';
```

Comment:
All these should import from `@/vibes/soul/`

Reply:
Thanks for the callout @jorgemoya! I'll open a PR that fixes this

Reply:
VIBES: https://github.com/makeswift/vibes/pull/412
Catalyst: https://github.com/bigcommerce/catalyst/pull/1766